package.json

```diff
@@ -60,16 +60,16 @@
     "./config-local": false
   },
   "dependencies": {
-    "@google-cloud/storage": "5.7.0",
```

Comment:
please verify cloud resource AWS and Azure, we've seen API breaks on new versions

Reply:
There are no noticeable breaking changes from 5.7.0 to 5.8.3